tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1145,6 +1145,10 @@ def _verify_profile_strategy(self, strategy):
       raise ValueError(
           ("profile_strategy '{}' is not supported. It should be one of {}"
           ).format(strategy, supported_profile_strategies()))
+    if strategy is "ImplicitBatchModeCompatible":
```

Comment:
change was rolled back automatically because here s/is/==/ 
please resubmit this with a new PR.